lib/update.rb

```diff
@@ -169,15 +169,17 @@ def self.prep_update
         filename = "https://api.github.com/repos/elanthia-online/lich-5/releases/latest"
         update_info = URI.parse(filename).open.read
 
-        JSON::parse(update_info).each { |entry|
+        JSON::parse(update_info).each { |entry, value|
           if entry.include? 'tag_name'
             @update_to = entry[1].sub('v', '')
```

Comment:
This still needs updating to use `value` so it should end up looking like:
```ruby
@update_to = value.sub('v', '')
```

Reply:
👍